ext-src/cpx.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/numa.cc b/src/numa.cc
+index d72c99e..16c903d 100644
+--- a/src/numa.cc
++++ b/src/numa.cc
+@@ -26,6 +26,7 @@ namespace mscclpp {
+ 
+ MSCCLPP_API_CPP int getDeviceNumaNode(int cudaDev) {
+   std::string busId = getBusId(cudaDev);
++  busId[busId.length() - 1] = '0';
```

Comment:
Is this always valid? It only replaces the last character. If `busId` ends in multiple digits, this may not do what you want.

Reply:
This is overriding the device function IDs in CPX mode. They are single digit.